packages/terra-core-docs/src/terra-dev-site/doc/section-header/example/FocusSectionHeader.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useRef } from 'react';
+import SectionHeader from 'terra-section-header';
+
+const FocusSectionHeader = () => {
+  const sectionHeader = useRef();
+
+  const sectionHeaderRef = (header) => {
+    sectionHeader.current = header;
+  };
+
+  const handleClick = () => {
+    sectionHeader.current.setFocus();
```

Comment:
```js
sectionHeader.current.focus();
``` 
We can also set focus on `sectionHeader` element like above code why do we need a handler to set Focus on component.?